rosdep/base.yaml

```diff
@@ -2177,6 +2177,24 @@ libopenscenegraph:
     portage:
       packages: [dev-games/openscenegraph]
   ubuntu: [openscenegraph, libopenscenegraph-dev]
+libopenvdb-dev:
+  arch: [openvdb]
+  debian: [libopenvdb-dev]
+  gentoo:
+    portage:
+      packages: [media-gfx/openvdb]
+  ubuntu: [libopenvdb-dev]
+libopenvdb:
+  arch: [openvdb]
+  debian:
+    jessie: [libopenvdb2.3]
+    sid: [libopenvdb3.2]
+  gentoo:
+    portage:
+      packages: [media-gfx/openvdb]
+  ubuntu:
+    trusty: [libopenvdb2.1]
```

Comment:
Please fill in the current platforms we support: missing are saucy, utopic, vivid, wily. And soon we'll add yakkety and zesty for Lunar which would be great too.